tests/sentry/api/endpoints/test_source_map_debug.py

```diff
@@ -69,6 +69,43 @@ def test_no_frame_given(self):
         )
         assert resp.data["detail"] == "Query parameter 'frame_idx' is required"
 
+    @with_feature("organizations:fix-source-map-cta")
+    def test_event_has_context_line(self):
+        event = self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "exception": {
+                    "values": [
+                        {
+                            "type": "Error",
+                            "stacktrace": {
+                                "frames": [
+                                    {
+                                        "abs_path": "https://app.example.com/static/js/main.fa8fe19f.js",
+                                        "filename": "/static/js/main.fa8fe19f.js",
+                                        "lineno": 1,
+                                        "colno": 39,
+                                        "context_line": "function foo() {",
+                                    }
+                                ]
+                            },
+                        },
+                    ]
+                },
+            },
+            project_id=self.project.id,
+        )
+
+        resp = self.get_success_response(
+            self.organization.slug,
+            self.project.slug,
+            event.event_id,
+            frame_idx=0,
+            exception_idx=0,
```

Comment:
Can we add a test when `frame_idx` and `exception_idx` goes out of index on the frame and exception?